scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController.java

```diff
@@ -0,0 +1,72 @@
+package org.cloudfoundry.autoscaler.scheduler.rest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.ApplicationScalingSchedules;
+import org.cloudfoundry.autoscaler.scheduler.service.ScheduleManager;
+import org.cloudfoundry.autoscaler.scheduler.util.error.InvalidDataException;
+import org.cloudfoundry.autoscaler.scheduler.util.error.ValidationErrorResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * 
+ *
+ */
+@RestController
+@RequestMapping(value = "/v2/schedules/{app_id}")
+public class ScheduleRestController {
+
+	@Autowired
+	private ValidationErrorResult validationErrorResult;
+	@Autowired
+	ScheduleManager scalingScheduleManager;
```

Comment:
just curious, if its type is ScheduleManager, why is its name scalingScheduleManager?